### DISCUSSION TITLE
[react-markdown] Warning: please pass a string as `children` (not: `[object Object]`)

### DISCUSSION THREAD

**Discussion by hecker** [remarkjs/remark#1161] [Q&A] [answered]
I currently get this error, I tried a lot of different things out but nothing really works. I am using ReactMarkdown to sync my Notion posts to my personal blog page.

Here are the two relevant code snippets: [lib/notion.ts](https://github.com/hecker/hecker.vc/blob/main/lib/notion.ts) and [app/(navbar)/blog/[slug]/page.tsx](https://github.com/hecker/hecker.vc/blob/main/app/(navbar)/blog/%5Bslug%5D/page.tsx).

I appreceate any help!

**Comment by wooorm** [comment, marked as answer]
Well, the error says it all. You pass an object. Not a string. What you call “`mdString`”, is not a string, but an object. You can either read the docs of the project that you are using (https://github.com/souvikinator/notion-to-md#readme), or place a `console.log` somewhere, to see that you get an object, and what you need to do instead
